src/ext/exe.rs

```diff
@@ -75,15 +76,29 @@ impl ExeMeta {
         Ok(())
     }
 
+    async fn write_binary(&self, data: &Bytes) -> Result<()> {
+        let dest_dir = self.cache_dir.as_path().join(&self.get_name());
+        fs::create_dir_all(&dest_dir).unwrap();
+        let mut file = File::create(&dest_dir.join(Path::new(&self.exe))).unwrap();
+        match file.write_all(&data) {
+            Err(err) => panic!("Error writing binary file: {}", err),
+            Ok(()) => Ok(()),
+        }
+    }
+
     async fn download(&self) -> Result<PathBuf> {
         log::info!("Command installing {} ...", self.get_name());
 
         let data = self
             .fetch_archive()
             .await
             .context(format!("Could not download {}", self.get_name()))?;
-        self.extract_archive(&data)
+        if self.name == "tailwindcss" {
```

Comment:
In order to make things more generic: could you instead of checking the name (which will only work for tailwindcss) check the file ending? Bascially `write_binary` could be called on line 67 instead of the bail!. 

And too keep things clear, rename `extract_archive` to something like `write_downloaded`